project/SimpleHTTPServer.scala

```diff
@@ -1,9 +1,10 @@
-import java.io.{ Closeable, File }
+import java.io.{Closeable, File}
```

Comment:
Was this an intentional reformat? Seems to violate our scalariform config (which I guess isn't applied to this project).